portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -33,9 +37,8 @@
  */
 @WebServlet("/my-form-handler")
 public class FormHandlerServlet extends HttpServlet {
+  private static final UserService userService = UserServiceFactory.getUserService();
   private static final String TEXT = "comment-text";
-  private static final String NAME = "commenter-name";
-  private static final String EMAIL = "commenter-email";
   private static final String IMAGE = "image";
 
   @Override
```

Comment:
add javadoc to doPost method

Reply:
iirc, this goes under the self-explanatory exception to javadoc.